common/travis/build.sh

```diff
@@ -6,6 +6,10 @@ if [ "$1" != "$2" ]; then
 	arch="-a $2"
 fi
 
+if [ "$3" = 1 ]; then
```

Comment:
Now that I look at it, this could have been just a check if they are the same arch, as is done for `$arch`. But having it controllable from the YAML file could be useful.